src/utils/data.js

```diff
@@ -1,8 +1,13 @@
 export let filterDataByUserAllocation = (datasets, allocations) => {
     let filteredDatasets = []
     datasets.forEach((dataset) => {
+        if(dataset.myallocation > 0){
+            filteredDatasets.push(dataset)
+            return
+        }
         allocations.forEach((allocation) => {
             if(allocation.nftAddress === dataset.nftaddress){
+                console.log(dataset)
```

Comment:
I'm seeing some console logs but it seems useful.
Let's leave this on here for now.